### PR TITLE
Newsletter 18: Matrix section and various release preparation

### PR DESCRIPTION
**Pull request by joepie91**
As requested on IRC :)

There are a few TODOs I haven't handled yet, since I'm not entirely sure what the correct procedure is. Date has been bumped to today (UTC+2).

### CODE REVIEW

File: content/2019-06-02-newsletter-18.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 +++
 title = "The Embedded Working Group Newsletter - 18"
-date = 2019-04-05
+date = 2019-06-02
```
Comment (therealprof) [inline content/2019-06-02-newsletter-18.md]:
```suggestion
date = 2019-07-02
```
Similarly the filename should be moved a month. ;)

File: content/2019-06-02-newsletter-18.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 draft = true
```
Comment (therealprof) [inline content/2019-06-02-newsletter-18.md]:
```suggestion
draft = false
```
Also let's activate this right away.

**Comment by therealprof** [comment]
Oh, and thanks for tackling this! Much appreciated.

File: content/2019-06-02-newsletter-18.md (hunk 1 above)
Comment (joepie91) [reply content/2019-06-02-newsletter-18.md]: ... wow. I probably shouldn't have done this late at night :)

File: content/2019-06-02-newsletter-18.md (hunk 2 above)
Comment (joepie91) [reply content/2019-06-02-newsletter-18.md]: Shouldn't the ecosystem crate numbers be updated first?

**Comment by therealprof** [comment]
Haven't checked for accuracy but there's a script in git@github.com:Disasm/rust-ewg-meta.git to fetch these details:

```
| Type                       | Status   | Count | Diff |
| :---                       | :-----   | :---- | :--- |
| [Peripheral Access Crates] | released | 22    | +4   |
| [HAL Impl Crates]          | released | 25    | +6   |
| [Board Support Crates]     | released | 18    | +5   |
| [Driver Crates Released]   | released | 25    | +4   |
| [Driver Crates WIP]        | WIP      | 67    | +4   |
| [no-std crates]            | released | 26    | +1   |
```

**Comment by joepie91** [comment]
Alright, I've applied the changes. I'll leave the discussion / next-newsletter links for someone else to fill in and uncomment post-release, as there's no next-newsletter stub in the repository yet :)

**Pull request merged**